packages/block-library/src/image/image-editing/use-transform-image.js

```diff
@@ -90,6 +91,15 @@ function useTransformState( { url, naturalWidth, naturalHeight } ) {
 		const el = new window.Image();
 		el.src = url;
 		el.onload = editImage;
+
+		const imgCrossOrigin = applyFilters(
+			'gutenberg.imageTransform.crossOrigin',
```

Comment:
Can this be "media.crossOrigin" or something like that? How will we make sure the "gutenberg" part is updated or removed when this lands in core? Is there a standard way to name filters?

Reply:
Is there a place for documentation of filters?

Reply:
Let's use `media.crossOrigin` and document it here https://github.com/WordPress/gutenberg/tree/004a32ec09d5b789be694d85725fe49cca385f08/docs/designers-developers/developers/filters 👍